examples/extensions/commands/custom_commands.rst

```diff
@@ -0,0 +1,174 @@
+.. _examples_extensions_custom_commands:
+
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands. These are some of the rules/tips to create custom commands:
+
+* All the custom commands must be located under the ``[YOUR_CONAN_HOME]/extensions/commands/`` folder.
+    * It can be created another folders layer to act like ``scope`` for the commands under it, e.g., ``[YOUR_CONAN_HOME]/extensions/commands/myscope/``.
+* Every custom command file must be named like ``cmd_[YOUR_CUSTOM_COMMAND_NAME].py``.
+* Any command can have sub-commands.
+* It can be performed different formatters to output any command result.
+* It is automatically injected one object instance of :ref:`ConanAPIV2 class<reference_python_api_conan_api_v2>` as first paremeter in the command function.
+
+Let's see some basic examples of custom commands:
+
+Simple "hello" command
+--------------------------
+
+This is the simplest command that runs a ``conan hello`` command and outputs ``Hello World!`` in your console.
```

Comment:
```suggestion
This is the simplest command that runs a ``conan hello`` command and outputs ``Hello World!`` in your console.
```
also a bit confusing wording - `simplest command that runs a command ...`, but there is just one command involved, right?